src/modules/aws_route53_hosted_zones@0.0.1/entity/resource_records_set.ts

```diff
@@ -0,0 +1,69 @@
+import {
+  Entity,
+  PrimaryGeneratedColumn,
+  Column,
+  ManyToOne,
+  JoinColumn,
+  AfterLoad,
+  AfterInsert,
+  AfterUpdate,
+} from 'typeorm'
+
+import { HostedZone } from './hosted_zone';
+
+export enum RecordType {
+  A = "A",
+  AAAA = "AAAA",
+  CAA = "CAA",
+  CNAME = "CNAME",
+  DS = "DS",
+  MX = "MX",
+  NAPTR = "NAPTR",
+  NS = "NS",
+  PTR = "PTR",
+  SOA = "SOA",
+  SPF = "SPF",
+  SRV = "SRV",
+  TXT = "TXT",
+}
+
+// TODO add unique constraint based on name + record type + parenthostedzone
+@Entity()
+export class ResourceRecordSet {
+  @PrimaryGeneratedColumn()
+  id?: number;
+
+  @Column({ nullable: true, })
+  name: string;
+
+  @Column({
+    type: 'enum',
+    enum: RecordType,
+  })
+  recordType: RecordType;
+
+  @Column()
+  records: string
```

Comment:
Had my doubts here. I did it because for some record types you can have multiple records having a multiline string.

Reply:
Can we make it singular? It definitely reads weird in the SQL in the test suite below, and anyone trying to use the multi-line multi-record record stuff will "get it" because they're already used to dealing with that weirdness with DNS, I think?

Maybe later on we can do something fancy with custom Postgres types here if/when more mainstream ORMs support it?